spec/factories/purpose_factories.rb

```diff
@@ -145,7 +145,7 @@
     target_type { 'Tube' }
   end
 
-  factory :tube_rack_purpose, class: TubeRack::Purpose do
+  factory :tube_rack_purpose, class: 'TubeRack::Purpose' do
```

Comment:
Why do these need to be quoted?

Reply:
A Rubocop suggestion. Essentially it allows the class to be lazy-loaded. This means you don't need to load in your entire app every time the factories are loaded. Makes for faster running of individual test files. At least in theory. In practice Sequencescape is such a spider's web that we probably end up loading in most of it anyway.